### PR TITLE
Add a dash at start of changelog message if not present

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: obal/data/modules/changelog.py [hunk 1]
```diff
@@ -50,6 +50,9 @@ def main():
 
     changed = False
 
+    if not changelog.starts_with('-'):
```
Comment (ekohl) [inline obal/data/modules/changelog.py]: Looks like a Rubyism
```suggestion
    if not changelog.startswith('-'):
```

**Comment by evgeni** [comment]
Won't this append a dash instead of prepending it?

**Comment by ekohl** [comment]
Sounds like this deserves tests :)

**Comment by ehelms** [comment]
Updated with a test update

**Pull request merged**